modules/bbmap/align/meta.yml

```diff
@@ -0,0 +1,51 @@
+name: bbmap_align
+description: write your description here
+keywords:
+  - align
+  - fasta
+  - genome
+  - reference
+tools:
+  - bbmap:
+      description: BBMap is a short read aligner, as well as various other bioinformatic tools.
+      homepage: https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/
+      documentation: https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/
+      tool_dev_url: None
+      doi: ""
+      licence: ['UC-LBL license (see package)']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
```

Comment:
Based on the description, would the input channel not be better named `fastq`?

Reply:
If that's what it usually is called, fine with me. I'll update.

Reply:
I don't actually know. But I personally would rather it be more explicit.